lib/autoload/active_record/associations/preloader/manual_has_many_through.rb

```diff
@@ -0,0 +1,5 @@
+class ActiveRecord::Associations::Preloader
+  class ManualHasManyThrough < HasMany
```

Comment:
The thing to figure out is if the through association is also populated when including a through association. If you just use the HasMany preloader then it only loads the extremes of the association.